2022q4/riscv.adoc

```diff
@@ -0,0 +1,42 @@
+=== FreeBSD/riscv64 Improvements
+
+Links: +
+link:https://wiki.freebsd.org/riscv[Wiki Homepage] URL: link:https://wiki.freebsd.org/riscv[https://wiki.freebsd.org/riscv] +
+
+Contact: Mitchell Horne <mhorne@FreeBSD.org>
+
+The FreeBSD/RISC-V project is providing support for running FreeBSD on the
+[RISC-V Instruction Set Architecture](https://riscv.org/).
+
+This quarter I resumed work on improvements to FreeBSD's RISC-V architecture
+support (riscv64). Work was focused was primarily on small bug-fixes and
+improvements, and tooling. A handful of known panics and bug reports were fixed
```

Comment:
i'd also write this as one sentence per line.
this doesn't have a big impact on one shots like the quarterly reports, but it's a good habit to keep for writing documentation. it makes diffs much easier to read